### PR TITLE
9.x-2.x Update dependencies and core compatibility

### PR DESCRIPTION
**Pull request by NikLP**
## Description

Updated composer.json to reflect updated dependencies
Updated info files to retain compatibility with D9+

## Motivation and Context

Module requires update for Drupal 9+ because of outdated dependencies

## How Has This Been Tested?

None yet

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] non breaking change (non-breaking change which is not issue related)
- [ ] Security related (updates which are security related)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by NikLP** [comment]
Would it be worth updating this to point to affinity-digital/omnipay at this point instead of drupal/omnipay? I've updated everything else except the module "name".

File: composer.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
     "description": "Omnipay Payment Gateways",
     "keywords": ["Drupal"],
     "license": "GPL-2.0+",
-    "homepage": "https://www.drupal.org/project/omnipay",
+    "homepage": "https://github.com/Affinity-Digital/omnipay",
```
Comment (SOMERSOFTAndrewCrook) [inline composer.json]: Please revert so that it is good for drupal.org

File: composer.json [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
     "minimum-stability": "dev",
     "support": {
-        "issues": "https://www.drupal.org/project/issues/omnipay",
-        "source": "http://cgit.drupalcode.org/omnipay"
+        "issues": "https://github.com/Affinity-Digital/omnipay/issues",
```
Comment (SOMERSOFTAndrewCrook) [inline composer.json]: Please revert so that it is good for drupal.org

File: composer.json [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
+        "source": "https://github.com/Affinity-Digital/omnipay"
```
Comment (SOMERSOFTAndrewCrook) [inline composer.json]: Please revert so that it is good for drupal.org

**Review by SOMERSOFTAndrewCrook — CHANGES_REQUESTED** [review]

**Pull request closed without merging**